src/host/ncp_host.cpp

```diff
@@ -250,6 +250,23 @@ void NcpHost::AddThreadEnabledStateChangedCallback(ThreadEnabledStateCallback aC
     OT_UNUSED_VARIABLE(aCallback);
 }
 
+#if OTBR_ENABLE_BACKBONE_ROUTER
+void NcpHost::SetBackboneRouterEnabled(bool aEnabled)
+{
+    mNcpSpinel.SetBackboneRouterEnabled(aEnabled);
+}
+
+void NcpHost::SetBackboneRouterMulticastListenerCallback(BackboneRouterMulticastListenerCallback aCallback)
+{
+    mNcpSpinel.SetBackboneRouterMulticastListenerCallback(aCallback);
+}
+
+void NcpHost::SetBackboneRouterStateChangedCallback(BackboneRouterStateChangedCallback aCallback)
+{
+    mNcpSpinel.SetBackboneRouterStateChangedCallback(aCallback);
+}
```

Comment:
For the std::function objects, either:
- Pass by value and use move constructor
or
- Pass by const reference